Support for SRP breaks compatibility with Unity 2018

Importing Zinnia to Unity versions below 2019 (I'm using 2018.4.5f1) causes the following compile error:

`Assets\Zinnia.Unity\Runtime\Process\Moment\MomentProcessor.cs(93,53): error CS0246: The type or namespace name 'ScriptableRenderContext' could not be found (are you missing a using directive or an assembly reference?)`

I believe ScriptableRenderContext and RenderPipelineManager were only introduced in Unity 2019.

--- a/Runtime/Process/Moment/MomentProcessor.cs
+++ b/Runtime/Process/Moment/MomentProcessor.cs
@@ -90,10 +90,12 @@
             }
         }
 
+#if UNITY_2019_1_OR_NEWER
         protected virtual void OnSrpCameraPreRender(ScriptableRenderContext context, Camera givenCamera)
         {
             Process();
         }
+#endif
 
         protected virtual void OnCameraPreRender(Camera givenCamera)
         {
@@ -115,7 +117,9 @@
                 case Moment.PreRender:
                     if (GraphicsSettings.renderPipelineAsset != null)
                     {
+#if UNITY_2019_1_OR_NEWER
                         RenderPipelineManager.beginCameraRendering -= OnSrpCameraPreRender;
+#endif
                     }
                     else
                     {
@@ -138,7 +142,11 @@
                 case Moment.PreRender:
                     if (GraphicsSettings.renderPipelineAsset != null)
                     {
+#if UNITY_2019_1_OR_NEWER
                         RenderPipelineManager.beginCameraRendering += OnSrpCameraPreRender;
+#else
+                        Debug.LogWarning("SRP is only supported on Unity 2019.1 or above");
+#endif
                     }
                     else
                     {

